expfmt/openmetrics_create.go

```diff
@@ -22,11 +22,25 @@ import (
 	"strconv"
 	"strings"
 
+	"google.golang.org/protobuf/types/known/timestamppb"
+
 	"github.com/prometheus/common/model"
 
 	dto "github.com/prometheus/client_model/go"
 )
 
+type toOpenMetrics struct {
+	withCreatedLines bool
+}
+
+type ToOpenMetricsOption func(*toOpenMetrics)
+
+func WithCreatedLines() ToOpenMetricsOption {
```

Comment:
Maybe add comment why this can be dangerous to use and e.g. we consider changing this? https://github.com/prometheus/prometheus/issues/13537

Reply:
Good point!